Zend/zend_globals.h

```diff
@@ -129,7 +135,7 @@ struct _zend_compiler_globals {
 
 	zend_stack delayed_oplines_stack;
 	HashTable *memoized_exprs;
-	int memoize_mode;
+	zend_memoize_mode memoize_mode;
```

Comment:
I can see a few stores of `CG(memoize_mode)` in `int` variables. We should adjust those to be consistent.

Reply:
Right, found two local variables and converted to the new enum.